Description: Relax em-http-request gem.
Author: Utkarsh Gupta <utkarsh@debian.org>
Bug-Debian: https://bugs.debian.org/959392
Bug-Debian: https://bugs.debian.org/967063

--- a/faye.gemspec
+++ b/faye.gemspec
@@ -22,7 +22,7 @@
             client_files
   
   s.add_dependency 'cookiejar', '>= 0.3.0'
-  s.add_dependency 'em-http-request', '>= 1.1.6'
+  s.add_dependency 'em-http-request', '>= 1.1.5'
   s.add_dependency 'eventmachine', '>= 0.12.0'
   s.add_dependency 'faye-websocket', '>= 0.11.0'
   s.add_dependency 'multi_json', '>= 1.0.0'
